src/core/varlink.c

```diff
@@ -14,12 +14,15 @@
 #include "unit.h"
 #include "user-util.h"
 #include "varlink.h"
+#include "varlink-internal.h"
+#include "varlink-io.systemd.Unit.h"
```

Comment:
not sorted

Reply:
these two lines are sorted to the best of my knowledge of the English alphabet